pkg/quantile/sparse.go

```diff
@@ -12,7 +12,7 @@ import (
 type Sketch struct {
 	sparseStore
 
-	Basic summary.Summary
+	Basic summary.Summary `json:"summary"`
```

Comment:
So we only get the summary in the serialized JSON, not the bins? We should at least comment to that effect.

Reply:
thanks! added a comment